### ISSUE TITLE
dark / light themes

### ISSUE THREAD

**Issue by blahah** [joeyklee/mozmap2015#9]
a mofo designer at mozfest (who was very complimentary) thought it might look cool with a dark/black background. We tried just changing the bg colour and it did look very nice - but lots of other colours (e.g. text) would need switching to make it work well. We could just have a little light-switch or a sun/moon icon that flips the theme?

**Comment by joeyklee** [comment]
Yup - I can add maybe a "colorsLight" and "colorsDark" key/value into the config.js - will play with it.

**Comment by blahah** [comment]
awesome - and we'll need something for the text and other colours

**Comment by joeyklee** [comment]
Style sheets are done - just need to connect it to a toggle in the view
